### PR TITLE
docs: add the CONTRIBUTING.md 

### LINKED ISSUE #54: Bullet points in README.md
### What
- "why facets-knowledge-panels" in README shows topics in paragraph style on the [GitHub page](https://openfoodfacts.github.io/facets-knowledge-panels/) instead of a bullet points.
- Also add CONTRIBUTING.md inside doc/how-to-guides can take some inspiration from [here](https://github.com/openfoodfacts/hunger-games/blob/master/CONTRIBUTING.md)

### Screenshots
image

### PR DESCRIPTION
**Pull request by Aju100**
Fixes #54 
- Here I have added CONTRIBUTING.md with the respective path.
- Also the fixing bullet points, its already been fixed on this PR https://github.com/openfoodfacts/facets-knowledge-panels/pull/52

cc @sumit-158

### CODE REVIEW

**Review by sumit-158 — CHANGES_REQUESTED** [review]
Great work @Aju100 👍🏼, just some small changes, and we are good to go.

File: docs/how-to-guides/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## Types:
+# wip - Work in Progress; long term work; mainstream changes;
+# feat - New Feature; future planned; non-mainstream changes;
+# bug - Bug Fixes
+# exp - Experimental; random experimental features;
+```
+
+- On Task Completion:
+
```
Comment (sumit-158) [inline docs/how-to-guides/CONTRIBUTING.md:52]: I would say we can put the link for "project setup" before On Task Completion, maybe the title can be  "setup project locally"

File: docs/how-to-guides/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+```bash
+## Committing and pushing your work
+
+# Ensure branch
```
Comment (sumit-158) [inline docs/how-to-guides/CONTRIBUTING.md:56]: Add "make checks" for format your code

**Review by sumit-158 — CHANGES_REQUESTED** [review]

**Comment by sumit-158** [comment]
@Aju100 what I mean by the bullet points in README is that if you check the [github page](https://openfoodfacts.github.io/facets-knowledge-panels/) for the documentation the "Why facets-knowledge-panels?" is not styled well here 
image
you have to change the items inside that section into bullet points.

**Comment by Aju100** [comment]
@sumit-158 here is the latest change output. Have a look
output

**Review by sumit-158 — APPROVED** [review]
LGTM

**Comment by sumit-158** [comment]
Thank you for the Pr :)

**Pull request merged**